set-integration-testing/src/integration-test/java/eu/europeana/api/set/integration/client/ClientConfigurationTest.java

```diff
@@ -11,19 +14,21 @@ public class ClientConfigurationTest {
     @Test
     public void clientConfiguration_loadProperties() {
         ClientConfiguration configuration = new ClientConfiguration(loadProperties());
-        assertTrue(StringUtils.isNotEmpty(configuration.getOauthRequestParams()));
-        assertTrue(StringUtils.isNotEmpty(configuration.getOauthRegularUserToken()));
+        assertTrue(StringUtils.isNotEmpty(configuration.getAccessToken()));
+        assertTrue(StringUtils.isNotEmpty(configuration.getApiKey()));
         assertTrue(StringUtils.isNotEmpty(configuration.getServiceUri()));
-        assertTrue(StringUtils.isNotEmpty(configuration.getOauthServiceUri()));
+        assertTrue(StringUtils.isNotEmpty(configuration.getAuthTokenEndpoitUri()));
+        assertNotNull(configuration.getAuthGrant());
+
     }
 
     private Properties loadProperties() {
         Properties properties = new Properties();
-        properties.put(ClientConfiguration.PROP_SET_SERVICE_URI, "service-ur-test");
-        properties.put(ClientConfiguration.PROP_SET_API_KEY, "test");
-        properties.put(ClientConfiguration.PROP_OAUTH_REGULAR_USER_TOKEN, "oauth-token");
-        properties.put(ClientConfiguration.PROP_OAUTH_SERVICE_URI, "outh-test");
-        properties.put(ClientConfiguration.PROP_OAUTH_REQUEST_PARAMS, "params-test");
+        properties.put(ClientConfiguration.PROP_SET_SERVICE_URI, "http://localhost/set");
+        properties.put(ClientConfiguration.CONFIG_APIKEY, "test");
+        properties.put(GrantConstants.access_token, "token_test");
```

Comment:
better add the constant to ClientConfiguration instead of creatign the dependency on sb3

Reply:
fixed